### PR TITLE
Better input variable naming for OpenAI (TF)

### PR DESCRIPTION
**Pull request by bhavika**
# What does this PR do?

Uses the @unpack_inputs decorator instead of input_processing for OpenAI (TF). See https://github.com/huggingface/transformers/issues/16051

Tagging @Rocketknight1 for review! 

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/master/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/master/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Review by gante — APPROVED** [review]
Thank you for this one as well :D Same deal as in the other PR, as soon as I get a positive answer for the two points below:
1. confirm that the tests pass (`RUN_SLOW=1 py.test -vv tests/[model_name]/test_modeling_tf_[model_name].py`)
2. address my other comment

**Comment by gante** [comment]
I've also made the same edit on the PR description :)

File: src/transformers/models/openai/modeling_tf_openai.py [hunk 1]
```diff
@@ -525,6 +507,7 @@ def __init__(self, config, *inputs, **kwargs):
         output_type=TFBaseModelOutput,
         config_class=_CONFIG_FOR_DOC,
     )
+    @unpack_inputs
```
[the remark this answers is not in the document]
Comment (bhavika) [reply src/transformers/models/openai/modeling_tf_openai.py]: Yes, thank you for catching this. Done in https://github.com/huggingface/transformers/pull/16129/commits/e6da3126115660472687a997d09ee4e45177fc75

**Comment by bhavika** [comment]
@gante Made changes as you suggested and I'm confirming that the tests pass for `RUN_SLOW=1 py.test -vv tests/openai/test_modeling_tf_openai.py`

**Pull request merged**